production/helm/loki/Chart.yaml

```
@@ -3,7 +3,7 @@ name: loki
 description: Helm chart for Grafana Loki and Grafana Enterprise Logs supporting monolithic, simple scalable, and microservices modes.
 type: application
 appVersion: 3.5.2
-version: 6.32.0
+version: 6.32.1
```

```suggestion
version: 6.32.1
```

revert this, not longer part of the PR